libcloud/dns/drivers/rackspace.py

```diff
@@ -131,6 +133,28 @@ def get_endpoint(self):
         return public_url
 
 
+class RackspacePTRRecord(object):
+    def __init__(self, id, ip, domain, driver, extra=None):
+        self.id = str(id) if id else None
+        self.ip = ip
+        self.type = RecordType.PTR
+        self.domain = domain
+        self.driver = driver
+        self.extra = extra or {}
+
+    def update(self, domain, extra=None):
+        return self.driver.ex_update_ptr_record(record=self, domain=domain,
+                                                extra=extra)
+
+    def delete(self):
+        return self.driver.ex_delete_ptr_record(record=self)
+
+    def __repr__(self):
+        return ('<%s: ip=%s, domain=%s, provider=%s ...>' %
```

Comment:
I only used the ```%``` interpolation because other objects in libcloud were doing so already.  I'm fine switching to ```.format()``` as that seems to be the preferred method these days.  Let me know.